src/System.IdentityModel.Tokens.Jwt/JwtSecurityToken.cs

```diff
@@ -100,36 +84,27 @@ public JwtSecurityToken(string jwtEncodedString)
         public JwtSecurityToken(JwtHeader header, JwtPayload payload, string rawHeader, string rawPayload, string rawSignature)
         {
             if (header == null)
-            {
-                LogHelper.Throw(string.Format(CultureInfo.InvariantCulture, LogMessages.IDX10000, GetType() + ": header"), typeof(ArgumentNullException), EventLevel.Verbose);
-            }
+                throw LogHelper.LogArgumentNullException("header");
 
             if (payload == null)
```

Comment:
This check is not needed. The check below for string.IsNullorEmpty is good enough.